room/runtime/src/test/java/androidx/room/SQLiteCopyOpenHelperTest.kt

```diff
@@ -221,6 +249,14 @@ class SQLiteCopyOpenHelperTest {
         }
     }
 
+    private fun gzipFile(input: File, output: File) {
+        input.inputStream().channel.use { inputChannel ->
```

Comment:
would have used `FileUtil` but it takes a `FileChannel` instead of a `WriteableByteChannel`

Reply:
If we move the tests to an android test then this won't be needed.